packages/components/organisms/f-checkout/src/services/addressGeocodingServiceApi.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+
+export default {
+    async getGeoLocation (url, postData, timeout, state) {
+        const authHeader = state.authToken && `Bearer ${state.authToken}`;
+
+        const config = {
+            headers: {
+                'Content-Type': 'application/json',
+                Authorization: authHeader
```

Comment:
Do you need to conditionally add this like you have done in `CheckoutServiceApi.js`?

Reply:
Ah so it wasn't originally set as a conditional before so I kept it like this when extracting - not sure why it hasn't been set though - maybe we could refactor it so it's similar to `CheckoutServiceApi`? @SachaZvetelman

Reply:
Some APIs require the `Authorization` header, and some don't and it's optional. **I think** that has to do with Guest users.
For the time being, just mimic what we had, since we know it's working.